packages/app/src/Feed/EventPublisher.ts

```diff
@@ -188,7 +188,16 @@ export default function useEventPublisher() {
         return await signEvent(ev);
       }
     },
-    zap: async (amount: number, author: HexKey, note?: HexKey, msg?: string) => {
+    /**
+     * Create a zap request event for a given target event/profile
+     * @param amount Millisats amout!
+     * @param author Author pubkey to tag in the zap
+     * @param note Note Id to tag in the zap
+     * @param msg Custom message to be included in the zap
+     * @param extraTags Any extra tags to include on the zap request event
+     * @returns
+     */
+    zap: async (amount: number, author: HexKey, note?: HexKey, msg?: string, extraTags?: Array<Array<string>>) => {
```

Comment:
The "e" and the "p" tag in this function are missing the primary hosting relay. That value is missing in your poll event though. Did the spec change since you made this PR?

<img width="885" alt="Screenshot 2023-04-08 at 10 55 11 AM" src="https://user-images.githubusercontent.com/3655410/230733718-e3b59489-dfb6-4354-94f8-77236955eec6.png">

Reply:
Most of those are optional, we will just implement the basic poll first

Reply:
For the poll event, I think that the `p` tag with the primary hosting relay is required. At least that's the current state of the nip-69 spec.

<img width="641" alt="Screenshot 2023-04-10 at 8 18 15 AM" src="https://user-images.githubusercontent.com/3655410/230919628-477fd939-1083-428d-923f-1e4f2c507380.png">

The nip-69 PR hasn't merged yet though, so I guess we could just ship without this stuff and update later to whatever ends up getting merged.